File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)

{
    int change;
    int quarters;
    int dimes;
    int nickels;
    int pennies;
    do
    {
        float dollar = get_float("Change: ");
        change=round(dollar*100);
    }
    while (change <= 0 );
    quarters = change / 25;
    dimes = (change % 25) / 10;
    nickels = ((change % 25) % 10 / 5 );
    pennies = ((change % 25) % 10) % 5;
    printf("%d\n", quarters + dimes + nickels + pennies);
    
}

File: pset6/cash/cash.py
from cs50 import get_float
change=0
owed=0
while 1:
    money = get_float("Change: ")
    if money>0:
        break
change=round(money*100)
while change>0:
    if change>=25:
        owed+=1
        change-=25
    elif change>=10:
        owed+=1
        change-=10
    elif change>=5:
        owed+=1
        change-=5
    elif change>=1:
        change-=1
        owed+=1
print(owed)